sys/linux/auto.txt

```diff
@@ -3772,10 +4557,30 @@ ioam6_genl_policy_ns_sc$auto_ioam6 [
 	IOAM6_ATTR_SC_NONE	nlattr[IOAM6_ATTR_SC_NONE, void]
 ] [varlen]
 
+iocb$auto_record {
+	aio_data	intptr
+	aio_key		int32
+	aio_rw_flags	int32
+	aio_lio_opcode	int16
+	aio_reqprio	int16
+	aio_fildes	int32
+	aio_buf		intptr
+	aio_nbytes	intptr
+	aio_offset	intptr
+	aio_reserved2	intptr
+	aio_flags	int32
+	aio_resfd	int32
+}
+
+iovec$auto_record {
+	iov_base	ptr[inout, void]
+	iov_len		int32
```

Comment:
Does the kernel has it as 32-bit type? Man says it's size_t, and kernel defines it as `__kernel_size_t`, which I would assume should be intptr.

Reply:
Should long and long long be both intptr? because `__kernel_size_t` is just a typedef to  `unsinged long`.

Reply:
You can check sizes of think on a small test C program running it in 64/32 modes.
For gcc/clang long is inptr, long long is 64 bits always.

Reply:
Done!